### PR TITLE
Some fix for configure script of s390x.

### PR DESCRIPTION
**Pull request by zhengxiaomei123**
**What type of PR is this?**
 /kind bug
/kind failing-test

**What does does this PR do / why we need it**:
Add same fix with #4206.
Add some fix for importing java imagestream.
**Which issue(s) this PR fixes**:

Fixes #?

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [ ] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Comment by rnapoles-rh** [comment]
@zhengxiaomei123 integration tests failed.

**Comment by dharmit** [comment]
@prietyc123 @mohammedzee1000 this PR is modifying the cluster configuration script for s390x architecture. But the 4.6 tests that are failing are being run on x86, right? What I'm observing in the test failures is that `odo catalog list services` command is failing in Operator Hub integration tests because it can't find any Operators installed on the cluster.

In this PR, I'm seeing that `KUBEADMIN_PASSWORD` value is being removed from configuration script of s390x: https://github.com/openshift/odo/pull/4317/files#diff-e24df2f816e33c2abe791ad856ac6f922a218705363d4f3176ec5e12050b5cb2L103-L106. But why would that cause failures in our integration tests that are running on x86? That should use `configure-installer-tests-cluster.sh` script, no?

**Comment by mohammedzee1000** [comment]
Interesting, that should not affect x86 indeed,

**Review by mohammedzee1000 — APPROVED** [review]
Looks good to me codewise
/lgtm

File: scripts/configure-installer-tests-cluster-s390x.sh [hunk 1]
```diff
@@ -73,9 +73,13 @@ sleep 5
 oc import-image nodejs:latest --from=registry.redhat.io/rhscl/nodejs-12-rhel7 --confirm -n openshift
 sleep 5
 oc annotate istag/nodejs:latest tags=builder -n openshift --overwrite
+oc delete istag java:8 -n openshift
+sleep 5
```
Comment (prietyc123) [inline scripts/configure-installer-tests-cluster-s390x.sh:77]: @zhengxiaomei123 Why do we have `sleep 5`? Wouldn't this slow down the test execution.

**Review by prietyc123 — CHANGES_REQUESTED** [review]

**Comment by rnapoles-rh** [comment]
@zhengxiaomei123 also wondering why the sleep 5 is needed.

File: scripts/configure-installer-tests-cluster-s390x.sh (hunk 1 above)
Comment (zhengxiaomei123) [reply scripts/configure-installer-tests-cluster-s390x.sh:77]: @prietyc123 @rnapoles-rh with Sleep 5 to confirm the image is here after importing and to confirm the image tag is deleted . We met error before without sleep 5. The error is not always happening but happening many times.

Comment (dharmit) [reply scripts/configure-installer-tests-cluster-s390x.sh:77]: @zhengxiaomei123 I agree with you point but @prietyc123 has a valid point here as well. One way to address both of your concerns could be to use the `oc import-image` command the way it's used here: https://github.com/openshift/odo/blob/f463d096610b9c88d2507d8481d0dd09b47ba650/tests/helper/helper_oc.go#L312-L317

More specifically, use `oc import-image` command with `--request-timeout 5m` flag. That should give us way more time than `sleep 5` would to import the image.

@prietyc123 WDYT?

Comment (prietyc123) [reply scripts/configure-installer-tests-cluster-s390x.sh:77]: > More specifically, use oc import-image command with --request-timeout 5m flag.

+1, I would really prefer to use `--request-timeout` than `sleep`.

Comment (zhengxiaomei123) [reply scripts/configure-installer-tests-cluster-s390x.sh]: @dharmit Thanks very much. I updated the code following your good points. @prietyc123 @mohammedzee1000 @rnapoles-rh Please help review again. Thanks.

**Comment by prietyc123** [comment]
Failing due to https://github.com/openshift/odo/issues/4301 .

/retest

**Review by rnapoles-rh — APPROVED** [review]

**Pull request merged**